README.md

```diff
@@ -54,6 +54,12 @@ Fantastic! The server is up-and-running, waiting for HTTP requests on `localhost
 curl http://localhost:8090/v2/network/information
 ```
 
+or to be accessed via CLI, e.g.:
+
+```
+docker run --network host inputoutput/cardano-wallet network information
```

Comment:
`run --rm`, otherwise things rapidly get out of hands ^^"

Reply:
fixed.